### PR TITLE
Invoke-FinOpsSchemaTransform and ConvertTo-FinOpsSchema + tests

### PR DESCRIPTION
**Pull request by flanakin**
## 🛠️ Description
- Add Invoke-FinOpsSchemaTransform command
- Update ConvertTo-FinOpsSchema command
- Add tests for both

### 🔬 How did you test this change?

> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [x] 💪 Unit tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Yes (required for `dev` PRs)
> - [x] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will cover in a future PR (feature branch PRs only)
> - [x] ❎ Not needed (small/internal change)

### CODE REVIEW

File: src/powershell/Tests/Unit/ConvertTo-FinOpsSchema.Tests.ps1
Comment (jamelachahbar) [inline src/powershell/Tests/Unit/ConvertTo-FinOpsSchema.Tests.ps1:1]: I am still getting the same error for operations involving Split-ResourceId function.
Only when removing these param values, it started working again.
        # [AllowNull]
        # [AllowEmptyString]

The rest of the scripts works as expected.

File: src/powershell/Public/Invoke-FinOpsSchemaTransform.ps1
Comment (jamelachahbar) [inline src/powershell/Public/Invoke-FinOpsSchemaTransform.ps1:1]: the parameter for the output file is named $OutputFile, which is clear, but the example uses -Destination, which could potentially confuse users.

I am getting this output. I will continue testing tomorrow as it is pretty late here.
"Length","LongLength","Rank","SyncRoot","IsReadOnly","IsFixedSize","IsSynchronized","Count"
"59436","59436","1","System.Object[]","False","True","False","59436"

File: src/powershell/Tests/Unit/ConvertTo-FinOpsSchema.Tests.ps1
Comment (flanakin) [reply src/powershell/Tests/Unit/ConvertTo-FinOpsSchema.Tests.ps1:1]: I noticed that, too. The fix I made didn't go in somehow 😓 I reapplied it in a separate PR to go into dev. I'll get that merged into this branch as well. Sorry!

File: src/powershell/Public/Invoke-FinOpsSchemaTransform.ps1
Comment (flanakin) [reply src/powershell/Public/Invoke-FinOpsSchemaTransform.ps1:1]: Doh! I was debating which way to go and guess I missed it. Nice catch!

File: src/powershell/Tests/Unit/ConvertTo-FinOpsSchema.Tests.ps1
Comment (flanakin) [reply src/powershell/Tests/Unit/ConvertTo-FinOpsSchema.Tests.ps1:1]: FYI - Confimed the fix in features/focus. Still need to sync this branch.

Comment (flanakin) [reply src/powershell/Tests/Unit/ConvertTo-FinOpsSchema.Tests.ps1:1]: FYI - I confirmed it in the features/focus branch. Need to sync this one.

File: src/powershell/Public/Invoke-FinOpsSchemaTransform.ps1
Comment (flanakin) [reply src/powershell/Public/Invoke-FinOpsSchemaTransform.ps1:1]: Fixed this one!

**Pull request merged**